lib/utils/dom.js

```diff
@@ -52,24 +55,38 @@ export function waitForChild(ele, selector) {
 	});
 }
 
-export function click(obj, button = 0) {
-	const evt = document.createEvent('MouseEvents');
-	evt.initMouseEvent('click', true, true, window.wrappedJSObject, 0, 1, 1, 1, 1, false, false, false, false, button, null);
-	obj.dispatchEvent(evt);
+export function click(obj: EventTarget, button: number = 0): void {
+	obj.dispatchEvent(new MouseEvent('click', {
```

Comment:
Tested in all browsers (+ with isProgrammaticEvent) and this behaves identically to the (_very_ deprecated) `initMouseEvent`